scripts/test-sim-backwards.sh

```diff
@@ -38,6 +38,8 @@ storj-sim -x --host $STORJ_NETWORK_HOST4 network test bash "$SCRIPTDIR"/test-bac
 
 make -C "$BRANCH_DIR" install-sim
 
+sed -i -e 's#/release/#/branch/#g' $STORJ_NETWORK_DIR/satellite/0/config.yaml
```

Comment:
This replacement looks weird. I'm wondering what lines is it replacing?

Reply:
its replacing anywhere that have "/release/" in the config file

Reply:
```
$ cat /var/folders/s_/n4czty1x4tj297495m66bcsc0000gp/T/tmp.XXXXXXXXXX.NBgZoieO/satellite/0/config.yaml | grep "/release/"
console.static-dir: /private/var/folders/s_/n4czty1x4tj297495m66bcsc0000gp/T/tmp.XXXXXXXXXX.NBgZoieO/release/web/satellite
mail.template-path: /private/var/folders/s_/n4czty1x4tj297495m66bcsc0000gp/T/tmp.XXXXXXXXXX.NBgZoieO/release/web/satellite/static/emails
marketing.static-dir: /private/var/folders/s_/n4czty1x4tj297495m66bcsc0000gp/T/tmp.XXXXXXXXXX.NBgZoieO/release/web/marketing```

Reply:
Let's add a comment that this renames the static dir paths.